@xen-orchestra/web/src/components/vm/dashboard/VmDashboardQuickInfo.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <VtsQuickInfoCard class="vm-dashboard-quick-info" :loading="!isReady">
+    <VtsQuickInfoColumn>
+      <VtsQuickInfoRow :label="$t('state')">
+        <template #value>
+          <span class="power-state">
+            <VtsIcon :accent="powerState.accent" :icon="powerState.icon" />
+            {{ powerState.text }}
+          </span>
+        </template>
+      </VtsQuickInfoRow>
+      <VtsQuickInfoRow :label="$t('ip-address')" :value="vm.mainIpAddress" />
+      <VtsQuickInfoRow :label="$t('created-on')" :value="installDateFormatted" />
+      <VtsQuickInfoRow :label="$t('started')" :value="relativeStartTime" />
+      <VtsQuickInfoRow :label="$t('host')">
+        <template #value>
+          <span v-if="host" class="host-name">
+            <UiObjectIcon size="medium" type="host" :state="hostPowerState" />
+            {{ host.name_label }}
+          </span>
+        </template>
```

Comment:
It is a complex element, so if it is not present, add none.
```suggestion
          <span v-else>
            {{ $t('none') }}
          </span>
        </template>
```